packages/api/src/store/db.ts

```diff
@@ -106,16 +108,32 @@ export class DB {
     )
   }
 
-  async query(query, ...params) {
+  queryWithOpts<T, I extends any[] = any[]>(
+    query: QueryConfig<I>,
+    opts: QueryOptions = {},
+  ): Promise<QueryResult<T>> {
+    if (opts.useReplica && this.replicaPool) {
```

Comment:
This changes from a default of `true` to a default of `false` for useReplica. I like it `true` — we should almost always be using replica queries except for rare cases where we must have up-to-date data (probably billing-related)

Reply:
Fixed